.github/test-spec.yml

```diff
@@ -584,7 +584,7 @@
   - "samples/event_manager_proxy/**/*"
   - "snippets/emulated-*/**/*"
   - "tests/subsys/event_manager_proxy/**/*"
-  - "tests/benchmarks/multicore/idle/**/*"
+  - "tests/benchmarks/multicore/**/**/*"
```

Comment:
I would like to have downstream test-low-level executed if changes to these tests are done.
I'm not sure if I understand how this file works...

Reply:
The whole scope is done at nightly only.

Reply:
OK, I will revert this and manually trigger execution of altered tests.